installation_docs/docker.md

```diff
@@ -15,8 +15,13 @@
 1. Fork current repository
 2. Clone the forked repository
     ```bash
-    git clone git@github.com:<username></username>/CircuitVerse.git
+    git clone git@github.com:<username></username>/CircuitVerse.git --recursive
     ```
+    - incase you missed using the `--recursive` during the cloning, then use
+        ```bash
+        git submodule update --init
+        ```
+        to clone the contents of submodule
```

Comment:
```suggestion
    - Use 'git submodule update --init' to get the contents of the submodule if you missed using the '--recursive' option while cloning the repository or if you have already done so.
```

Reply:
@Arnabdaz just update the suggested content in other places, we are GTG to merge this